hawc/apps/epiv2/api.py

```diff
@@ -34,7 +34,7 @@ def export(self, request, pk):
         return Response(exporter.build_export())
 
 
-class Design(EditPermissionsCheckMixin, AssessmentEditViewset):
+class DesignViewset(EditPermissionsCheckMixin, AssessmentEditViewset):
```

Comment:
I modified this on all the ones I touched (DesignViewset -> DesignViewSet) but not on existing untouched ones (AssessmentEditViewset, etc.) since the other PR seems to cover those.